pkg/app/piped/planner/cloudrun/cloudrun.go

```diff
@@ -55,22 +55,23 @@ func (p *Planner) Plan(ctx context.Context, in planner.Input) (out planner.Outpu
 	}
 
 	// Determine application version from the manifest.
-	if version, e := p.determineVersion(ds.AppDir, cfg.Input.ServiceManifestFile); e == nil {
-		out.Version = version
-	} else {
+	if version, e := p.determineVersion(ds.AppDir, cfg.Input.ServiceManifestFile); e != nil {
```

Comment:
Let`s follow the go practice and this change is just refactoring, right?
```suggestion
	if version, err := p.determineVersion(ds.AppDir, cfg.Input.ServiceManifestFile); err != nil {
```

Reply:
Sorry for not make it clear, but I want this error to be e instead of err, to avoid shadowing with err defined in function signature 👀

Reply:
Sorry, I could not catch your point.
In this way, the err scope is restricted to if statement, right?

Reply:
I guess that `:` is for "version" variable, and err still the outer one. Don't want to take risk there so better to make it clear that it's a different one and be sure we don't return it by return statement.

Reply:
fyi, we did the same in case of kubernetes planner

ref https://github.com/pipe-cd/pipecd/blob/master/pkg/app/piped/planner/kubernetes/kubernetes.go#L93

Reply:
Umm, I guess it's common if there is no specific thing.

> fyi, we did the same in case of kubernetes planner

The same could be said here.

Reply:
Okay, I got your point, and let's keep it.
Thanks.

Reply:
Thank you 🙏